source/index-management/index-management.rst

```diff
@@ -414,6 +414,10 @@ Index View API
      * For drivers that cannot make IndexView iterable, they MUST implement this method to
      * return a list of indexes. In the case of async drivers, this MAY return a Future<Cursor>
      *  or language/implementation equivalent.
+     * 
+     *  If drivers are unable to make the IndexView iterable, they MAY opt to provide the options for 
+     *  listing search indexes via the `list` method instead of the `Collection.listSearchIndexes` method.
+
```

Comment:
After discussion with @dariakp on slack, we decided that a more consistent API is to include the `name` in an options object.  There were two reasons for this change:

- It's odd to have a set of search index management methods that have a name as the first parameter and an optional second parameter in others.  It makes more sense to include an optional name in an "options" object.
- This is more closely aligned with the regular index management API.

Reply:
I understand this was already allowed, this is how [it is already implemented in PHP](https://github.com/mongodb/mongo-php-library/blob/9b6d5a16411c07a52bae634f37d40bf193f30097/src/Collection.php#L400).